app/plugin.go

```diff
@@ -394,3 +398,48 @@ func (a *App) GetPlugins() (*model.PluginsResponse, *model.AppError) {
 
 	return resp, nil
 }
+
+// notifyPluginEnabled notifies web clients for all cluster peers if
```

Comment:
Can you go into some extra detail on the thinking behind this method, especially our discussion yesterday on why we can't just notify our own clients (e.g. because they could connect to another host that doesn't have the bundle)? 

Also, could you describe why this is guaranteed to converge, e.g. why two clients won't end up checking each other and skipping the update, leaving no one left to do so?